### ISSUE TITLE
Unable to execute

### ISSUE THREAD

**Issue by devlsh** [ubient/laravel-vapor-action#6]
```
 checking if debug is enabled... no
checking if zts is enabled... no
checking for gawk... no
checking for nawk... no
checking for awk... awk
checking if awk is broken... no
checking whether to enable multibyte string support... yes, shared
checking whether to enable multibyte regex support (requires oniguruma)... yes
checking for oniguruma... no
configure: error: Package requirements (oniguruma) were not met:

Package 'oniguruma', required by 'virtual:world', not found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables ONIG_CFLAGS
and ONIG_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
The command '/bin/sh -c apk add libxml2-dev &&     docker-php-ext-install bcmath xml tokenizer mbstring' returned a non-zero code: 1

##[error]Docker build failed with exit code 1
```

**Comment by devlsh** [comment]
Using:

```
jobs:
  build:
    name: Build
    runs-on: ubuntu-latest
    steps:
      - name: Clone Repo
        uses: actions/checkout@v2-beta
      - name: Deploy to Production
        uses: ubient/laravel-vapor-action@master
        env:
          VAPOR_API_TOKEN: ${{ secrets.VAPOR_API_TOKEN }}
        with:
          args: deploy production
```

**Comment by claudiodekker** [comment]
Hi, thank you for opening an issue & describing your configuration. 

A PR that should fix this issue was just merged (#5) & released. Let me know if your problem still occurs, otherwise feel free to close this issue!
